network-api/networkapi/wagtailpages/factory/publication.py

```diff
@@ -10,14 +14,37 @@
 
 
 class PublicationPageFactory(PageFactory):
-    title = Faker('text', max_nb_chars=255)
+    title = Faker('text', max_nb_chars=120)
 
     class Meta:
         model = PublicationPage
 
 
+class ArticlePageFactory(PageFactory):
+    title = Faker('text', max_nb_chars=120)
+
+    class Meta:
+        model = ArticlePage
+
+
 def generate(seed):
     reseed(seed)
     home_page = get_homepage()
+    """
+    Ceate a couple scenarios that will be best for testing: 
+    * A PublicationPage with several child ArticlePages
+    * A PublicationPage with child PublicationPages, each of which has their own ArticlePages
+        * perhaps nested at random levels of depth?
```

Comment:
i had a couple thoughts for ways to make this more random and played around with making some separate functions

in the end I decided this was more of a KISS scenario, something we can revisit when models are fleshed out and we have a sense of edge cases to represent